### PR TITLE
WIP: translate strings (#288)

### PR DESCRIPTION
**Pull request by RandyMcMillan**
Some guidance on this would be appreciated until I get comfortable with this.

### CODE REVIEW

**Comment by jonatack** [comment]
The PR description in #287 looks like helpful guidance for this.

(PS - don't  put @-prefixed usernames in commits and PR descriptions; the latter because usernames in the description are copied into the merge commit by the merge script. This can cause endless annoying notifications for those concerned. An update to the merge script now warns maintainers if a merge message contains a @username, but it's better not to add them to begin with. For more, see https://jonatack.github.io/articles/how-to-contribute-pull-requests-to-bitcoin-core#pr-descriptions.)

**Comment by hebasto** [comment]
@RandyMcMillan 

Thanks for working on this!

> The PR description in #287 looks like helpful guidance for this.

Indeed, for cases in this PR the translator comments should be used.

To ensure, that they are working properly, you could use `make -C src translate`.

Also for such changes I'd prefer a pr with a wide scope :)

**Comment by hebasto** [comment]
@RandyMcMillan 

If your goal is resolving #288, you could look into #220 where some disambiguation strings were introduced, and they should be replaced with translator comments.

File: src/qt/guiutil.cpp [hunk 1]
```diff
@@ -710,7 +710,7 @@ QString formatServicesStr(quint64 mask)
     if (strList.size())
         return strList.join(" & ");
     else
-        return QObject::tr("None");
+        return QObject::tr("None", "absence of value");
```
Comment (hebasto) [inline src/qt/guiutil.cpp:713]: Our goal is to replace useless (in most cases) disambiguation strings (the second argument in `tr` call) with translator comments:
```suggestion
        //: Absence of value.
        return QObject::tr("None");
```

**Comment by RandyMcMillan** [comment]
NOTE: Currently I am just trying to get an idea of how much work needs to be done.

**Comment by hebasto** [comment]
In its current state this PR does not follow our the most recent translation policy, as we are not using the second argument of the [`QObject:tr`](https://doc.qt.io/qt-5/qobject.html#tr) function.

#311 is a good example of the mentioned approach.

Closing for now.

**Pull request closed without merging**